# Detected object message as reported by vehicle.

std_msgs/Header header

float64 front_object_distance_low_res    # Measured distance to nearest front object in m
float64 front_object_distance_high_res   # Measured distance to nearest front object in m (high res)
